### ISSUE TITLE
Question about 0a 0b 0c 

### ISSUE THREAD

**Issue by luong-komorebi** [zdharma/zplugin-configs#14]
Hi there. First of all, thank you for your awesome work with zplugin. I have been using it for a week. Now I am curious and want to look for ways to make the config more maintainable.

In this particular one from Nicholas Serrano :
https://github.com/zdharma/zplugin-configs/blob/master/NICHOLAS85/.zshrc

I love how the functions are mapped. However, later I found that they are call with '0a, 0b, 0c'. 
https://github.com/zdharma/zplugin-configs/blob/master/NICHOLAS85/.zshrc#L50-L54

Do you have any ideas what those mean? Thank you for your time.

**Comment by DundarGoc** [comment]
From the News section in Zplugin:

  - Turbo mode, i.e. the `wait''` ice-mode now supports a suffix – the letter `a`, `b`
    or `c`. The meaning is illustrated by the following example:

    ```zsh
    zplugin ice wait"0b" as"command" pick"wd.sh" atinit"echo Firing 1" lucid
    zplugin light mfaerevaag/wd
    zplugin ice wait"0a" as"command" pick"wd.sh" atinit"echo Firing 2" lucid
    zplugin light mfaerevaag/wd

    # The output
    Firing 2
    Firing 1
    ```

    As it can be seen, the second plugin has been loaded first. That's because there
    are now three sub-slots (the `a`, `b` and `c`) in which the plugin/snippet loadings
    can be put into. Plugins from the same time-slot with suffix `a` will be loaded
    before plugins with suffix `b`, etc.

    In other words, instead of `wait'1'` you can enter `wait'1a'`,
    `wait'1b'` and `wait'1c'` – to this way **impose order** on the loadings
    **regardless of the order of `zplugin` commands**.

**Comment by luong-komorebi** [comment]
Thank you so much @DundarGoc . I could not find information about it in zplugin readme :D
